src/overlays/actors/ovl_En_Syateki_Niw/z_en_syateki_niw.h

```diff
@@ -28,46 +28,51 @@ typedef struct EnSyatekiNiw {
     /* 0x0190 */ Vec3s jointTable[16];
     /* 0x01F0 */ Vec3s morphTable[16];
     /* 0x0250 */ EnSyatekiNiwActionFunc actionFunc;
-    /* 0x0254 */ s16 unk_254;
-    /* 0x0256 */ s16 unk_256;
-    /* 0x0258 */ s16 unk_258;
-    /* 0x025A */ s16 unk_25A;
-    /* 0x025C */ s16 unk_25C;
-    /* 0x025E */ s16 unk_25E;
-    /* 0x0260 */ s16 unk_260;
-    /* 0x0262 */ s16 unk_262;
-    /* 0x0264 */ f32 unk_264;
-    /* 0x0268 */ f32 unk_268;
-    /* 0x026C */ f32 unk_26C;
+    /* 0x0254 */ s16 peckTimer;
+    /* 0x0256 */ s16 timer1; // sometimes set alongside peckTimer, never read on its own.
+    /* 0x0258 */ s16 flapTimer;
+    /* 0x025A */ s16 archeryTimer;
+    /* 0x025C */ s16 hopTimer;
+    /* 0x025E */ s16 movementTimer;
+    /* 0x0260 */ s16 sootTimer; // cuccoo is covereed in soot.
+    /* 0x0262 */ s16 cluckTimer;
+    /* 0x0264 */ f32 headRotXTarget;
+    /* 0x0268 */ f32 wingRightRotXTarget;
+    /* 0x026C */ f32 wingLeftRotXTarget;
     /* 0x0270 */ char unk_270[0x8];
-    /* 0x0278 */ f32 unk_278;
-    /* 0x027C */ f32 unk_27C;
-    /* 0x0284 */ f32 unk_280;
-    /* 0x0280 */ f32 unk_284;
-    /* 0x0288 */ f32 unk_288;
-    /* 0x028C */ s16 unk_28C;
-    /* 0x028E */ s16 unk_28E;
-    /* 0x0290 */ s16 unk_290;
-    /* 0x0292 */ s16 unk_292;
-    /* 0x0294 */ s16 unk_294;
-    /* 0x0296 */ s16 unk_296;
-    /* 0x0298 */ s16 unk_298;
-    /* 0x029C */ s16 unk_29A;
-    /* 0x029C */ s16 unk_29C;
-    /* 0x029E */ s16 unk_29E;
-    /* 0x02A0 */ s16 unk_2A0;
-    /* 0x02A4 */ Vec3f unk_2A4;
-    /* 0x02B0 */ Vec3f unk_2B0;
-    /* 0x02BC */ Vec3f unk_2BC;
-    /* 0x02C8 */ Vec3f unk_2C8;
-    /* 0x02D4 */ f32 unk_2D4;
-    /* 0x02D8 */ f32 unk_2D8;
-    /* 0x02DC */ Vec3f unk_2DC;
-    /* 0x02E8 */ Vec3f unk_2E8;
-    /* 0x02F4 */ f32 unk_2F4;
-    /* 0x02F8 */ u8 unk_2F8;
+    /* 0x0278 */ f32 wingRightRotYTarget;
+    /* 0x027C */ f32 wingRightRotZTarget;
+    /* 0x0284 */ f32 wingLeftRotYTarget;
+    /* 0x0280 */ f32 wingLeftRotZTarget;
+    /* 0x0288 */ f32 unkArcheryFloat; // set, but never used. another rot target?
+    /* 0x028C */ s16 lifetime; // always ticks up, never read
+    /* 0x028E */ s16 headRotXState;
+    /* 0x0290 */ s16 unk_290; // set to 0, never read.
+    /* 0x0292 */ s16 wingRotState;
```

Comment:
```suggestion
    /* 0x0292 */ s16 wingsRotState;
```
sounds a bit better to me